modules/apps/transfer/client/cli/tx.go

```diff
@@ -34,11 +31,9 @@ func NewTransferTxCmd() *cobra.Command {
 		Use:   "transfer [src-port] [src-channel] [receiver] [amount]",
 		Short: "Transfer a fungible token through IBC",
 		Long: strings.TrimSpace(`Transfer a fungible token through IBC. Timeouts can be specified
-as absolute or relative using the "absolute-timeouts" flag. Timeout height can be set by passing in the height string
-in the form {revision}-{height} using the "packet-timeout-height" flag. Relative timeout height is added to the block
-height queried from the latest consensus state corresponding to the counterparty channel. Relative timeout timestamp 
-is added to the greater value of the local clock time and the block timestamp queried from the latest consensus state 
-corresponding to the counterparty channel. Any timeout set to 0 is disabled.`),
+as absolute using the "absolute-timeouts" flag. Timeout height can be set by passing in the height string
+in the form {revision}-{height} using the "packet-timeout-height" flag. Note, relative timeout height is not supported. Relative timeout timestamp 
```

Comment:
Just a nit: it looks like in the other CLIs we enclose the flags between `{}`. Not sure if this affects the formatting of the flag on the command line...

```suggestion
in the form {revision}-{height} using the {packet-timeout-height} flag. Note, relative timeout height is not supported. Relative timeout timestamp 
```